strapi/strapi
# Data Transfer Capabilities

### What does it do?

Data Transfer is the second iteration of Strapi's take on data management.
It includes data transfer capabilities between two strapi instances, either local (current directory) or remote (running app).

### How to test it?

- Start a Strapi instance on any host or port. Get the admin full admin URL from this instance (`REMOTE_URL`)
- Go to `REMOTE_URL`, then settings, then transfer token, and generate a new one (`REMOTE_TOKEN`)
- Open a terminal session on another Strapi instance that shares the same structure (`LOCAL_STRAPI`).
- Run the following command to transfer data from `LOCAL_STRAPI` to `REMOTE_STRAPI`:
  - `yarn strapi transfer --to REMOTE_URL --to-token REMOTE_TOKEN --force`
- Alternatively, you can use the CLI prompts to guide you through the transfer
  - `yarn strapi transfer --to REMOTE_URL`
- The transfer progress should be displayed on your terminal and you should see a summary report about what's been transferred upon completion.
